### ISSUE TITLE
baseline model failures from schedule limits

### ISSUE THREAD

**Issue by mdahlhausen** [NREL/openstudio-standards#821]
From Matt Steen:
I've had two of my recent model's baselines fail with this message. looks like an orphaned schedule in the SR1 model with bad limits:
`ERROR The run failed with the following severe or fatal errors: ProcessScheduleInput: Schedule="HOT WATER LOOP TEMP - 180F" has values outside its Schedule Type (TEMPERATURE) range    Minimum should be >=-50.000 and Maximum should be <=50.000\nProcessScheduleInput: Preceding Errors cause termination.`

```
OS:ScheduleTypeLimits,
  {b9b4b60b-4381-44b4-a187-efa81fe54889}, !- Handle
  Temperature,                            !- Name
  -50,                                    !- Lower Limit Value
  50,                                     !- Upper Limit Value
  Continuous,                             !- Numeric Type
  Temperature;                            !- Unit Type
```

**Comment by eringold** [comment]
I ran into this and if I recall correctly, the standards-created schedule was picking up whatever ScheduleTypeLimits object named "Temperature" it found in the model. For whatever reason, the existing one happened to be limited to 50.

**Comment by MatthewSteen** [comment]
Changing the name of ScheduleTypeLimits object directly in the proposed OSM seems to fix the error. Search for the following in the proposed OSM and rename or delete the object after confirming its handle isn't referenced by another object.

```
  Temperature,                            !- Name
  -50,                                    !- Lower Limit Value
```

**Comment by asparke2** [comment]
@MatthewSteen  @mdahlhausen do we have an .osm to replicate this issue with?  I can probably figure it out without one, but testing a fix will be hard.

**Comment by MatthewSteen** [comment]
@asparke2 I don't have one handy. but if you add this object to a model it should fail when creating a baseline. I haven't tracked down why our proposed models have this unused schedule. We are using the standards to automate space type assignments, which could be part of it.

```
OS:ScheduleTypeLimits,
  {b9b4b60b-4381-44b4-a187-efa81fe54889}, !- Handle
  Temperature,                            !- Name
  -50,                                    !- Lower Limit Value
  50,                                     !- Upper Limit Value
  Continuous,                             !- Numeric Type
  Temperature;
```

**Comment by mdahlhausen** [comment]
Fixed with #877
